common/config/config.go

```diff
@@ -99,6 +99,20 @@ type Dendrite struct {
 		// If set disables new users from registering (except via shared
 		// secrets)
 		RegistrationDisabled bool `yaml:"registration_disabled"`
+		// Perspective keyservers, to use as a backup when direct key fetch
+		// requests don't succeed
```

Comment:
General query: does the spec state when to use perspective servers vs direct fetches? If not, presumably servers have the right to pick and choose a combination. Would it ideally make sense to sort servers by uptime/latency/etc and then optimistically try them first?

Reply:
I don't think the spec states specifically. In our case we attempt direct fetching first, followed by the perspectives in the order they are supplied.